deployment/docker-compose.prod.yml

```diff
@@ -37,14 +40,31 @@ services:
       - NS_MINIO_SECRET_KEY=supersecurepassword
       - NS_LOG_LEVEL=40
     command: [ "gunicorn", "-w 4", "-b 0.0.0.0:8080", "nunaserver.wsgi:app" ]
-
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost:8080/health"]
```

Comment:
`curl` is not part of the nunaweb-server image as of now. If we're using it, then we should add it to the Dockerfile build.

Reply:
curl is preinstalled